manifests/php.yml

```diff
@@ -181,6 +181,7 @@ tests/:
       Test_StandardizationBlockMode: missing_feature
     test_remote_config_rule_changes.py:
       Test_BlockingActionChangesWithRemoteConfig: missing_feature
+      Test_UpdateRuleFileWithRemoteConfig: missing_feature
```

Comment:
Should work since `v0.8.0` if you want to update it directly.

Reply:
I'll do it. Waiting for more feedbacks...

Reply:
done